backend/src/utils/db.py

```
@@ -0,0 +1,216 @@
+        # Get rugs avoided
+        rugs_avoided = get_rugs_avoided()
+
+        return {
+            "totalProfit": round(total_profit_sol, 4),
+            "totalTrades": total_buys + total_sells,
+            "totalBuys": total_buys,
+            "totalSells": total_sells,
+            "successRate": round(success_rate, 2),
+            "rugsAvoided": rugs_avoided
+        }
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)